starter-code/javascript/chronometer.js

```diff
@@ -8,19 +8,19 @@ class Chronometer {
   startClick(callback) {
     this.isRunning = true;
     this.intervalId = setInterval(() => {
-      this.currentTime += 1 ;
+      this.currentTime += 1;
       console.log(this.currentTime);
       callback();
-      }, 1000);
-    }
+    }, 1000);
```

Comment:
Adding the callback here breaks the jasmine test because Jasmine doesn't send a callback function when she tests the startClick function, in order to fix it, you could just test if the callback is defined or not in order to execute the function.

 ```
 startClick(callback) {
    this.isRunning = true;
    this.intervalId = setInterval(() => {
      this.currentTime += 1;
      console.log(this.currentTime);
      if (callback) {
        callback();
      }
    }, 1000);
  }
```